From: =?UTF-8?q?Ulrich=20=C3=96lmann?= <u.oelmann@pengutronix.de>
Date: Thu, 14 Aug 2025 12:45:13 +0200
Subject: [PATCH] arm64: dts: imx8mp-skov: disable caam job ring 2
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Job ring 2 is used by OP-TEE in the secure world and therefore unavailable to
the Linux kernel which runs in the normal world.

Upstream-Status: Pending

Signed-off-by: Ulrich Ölmann <u.oelmann@pengutronix.de>
---
 arch/arm64/boot/dts/freescale/imx8mp-skov-reva.dtsi | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/imx8mp-skov-reva.dtsi b/arch/arm64/boot/dts/freescale/imx8mp-skov-reva.dtsi
index 88245efd7056..5453e590f64f 100644
--- a/arch/arm64/boot/dts/freescale/imx8mp-skov-reva.dtsi
+++ b/arch/arm64/boot/dts/freescale/imx8mp-skov-reva.dtsi
@@ -459,6 +459,11 @@ &pwm4 {
 	pinctrl-0 = <&pinctrl_pwm4>;
 };
 
+&sec_jr2 {
+	/* used by OP-TEE */
+	status = "disabled";
+};
+
 &uart1 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_uart1>;
